neo3/api/noderpc.py

```diff
@@ -310,20 +313,35 @@ def from_json(cls, json: dict):
 
 
 @dataclass
-class AppliationLogResponse:
-    tx_id: types.UInt256
+class TransactionApplicationLogResponse:
```

Comment:
Could we add some in-line docs?

Reply:
You think it is not self-explanatory now?

Reply:
Yes, it is. Thanks!